src/OpenLoco/src/World/CompanyManager.cpp

```diff
@@ -300,7 +300,26 @@ namespace OpenLoco::CompanyManager
         // Original network logic removed
         auto& gameState = getGameState();
         gameState.flags |= GameStateFlags::preferredOwnerName;
-        auto competitorId = selectNewCompetitor(-1);
+
+        // Any preference with respect to owner face?
+        int8_t competitorId = -1;
+        if (Config::get().usePreferredOwnerFace)
+        {
+            auto& preferredFace = Config::get().preferredOwnerFace;
+            if (ObjectManager::load(preferredFace))
+            {
+                auto handle = ObjectManager::findObjectHandle(preferredFace);
+                if (handle)
+                {
+                    competitorId = (*handle).id;
+                }
+            }
+        }
```

Comment:
This feels a little redundant. Perhaps this is something that `selectNewCompetitor` could handle. However, that function is not implemented, and a cursory look at IDA makes that look way out of scope for this PR.